Lib/test/test_statistics.py

```diff
@@ -1004,6 +1004,9 @@ def test_nan(self):
             x = statistics._convert(nan, type(nan))
             self.assertTrue(_nan_equal(x, nan))
 
+    def test_raise_type_error(self):
+        self.assertRaises(TypeError, statistics._convert, None, float)
```

Comment:
It's not very good practice to test the internal implementation of a module. Would it be possible to write a test or improve another one to exert this code path without actually depending on the private names?

Reply:
I am not sure that this is possible if we want to hit the 4 missed statements that this PR covers.

The `test_statistics` module defines `TestCase` classes for every defined function of the `statistics` module, even for those defined as private. For instance, the `_convert` function has already 6 unit tests that invoke it directly.

Reply:
Indeed, this is just a small expansion of an existing test suite for these module internals.

IMO given the existing tests this is reasonable and relevant.